src/styles/index.scss

```diff
@@ -1,3 +1,87 @@
-body {
+.page_body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    transform-origin: bottom center;
+    animation: rotate-minute 3600s infinite steps(60);
```

Comment:
It's better to keep all the animation properties together. Consider moving `animation` property to the end of the block for better readability and maintainability.